04 Abstraction Exercises Part 1/Public Vehicle/src/main/java/com/project/martinez/PublicVehicle.java

```diff
@@ -0,0 +1,22 @@
+package com.project.martinez;
+
+public class PublicVehicle {
+    private double fare;
```

Comment:
Fare field is not necessary since it will be computed. Hint: this class should have a field that has the base fare of for all the vehicles.